src/screens/Routes/Routes.ios.js

```diff
@@ -247,25 +243,27 @@ const routesForBottomSheetStack = {
   [Routes.STACK]: Stack,
   [Routes.CHANGE_WALLET_SHEET]: {
     navigationOptions: {
-      allowsDragToDismiss: true,
       backgroundOpacity: 0.6,
       customStack: true,
       springDamping: 1,
       transitionDuration: 0.25,
     },
     screen: ChangeWalletSheet,
   },
-  [Routes.RECEIVE_MODAL]: withCustomStack(ReceiveModal),
-  [Routes.SETTINGS_MODAL]: withCustomStack(SettingsModal),
+  [Routes.CONFIRM_REQUEST]: {
+    navigationOptions: {
+      backgroundOpacity: 1,
+      customStack: true,
+      springDamping: 1,
+      transitionDuration: 0.25,
+    },
+    screen: TransactionConfirmationScreen,
+  },
   [Routes.EXPANDED_ASSET_SHEET]: {
     navigationOptions: {
-      allowsDragToDismiss: true,
-      allowsTapToDismiss: true,
       backgroundOpacity: 0.7,
-      blocksBackgroundTouches: true,
       cornerRadius: 24,
       customStack: true,
-      gestureEnabled: true,
```

Comment:
These changes look unrelated to the PR. Just checking that you want to include jt

Reply:
Ah yeah that was intentional, just saw it while I was touching the file

99% sure that doesn’t do anything in the slack sheets, no difference when I tested